example/convertingproxy/benchmark.md

```diff
@@ -0,0 +1,77 @@
+# Benchmark
+
+## Benchmark With Socks SuperProxy
+
+SuperProxy is a VPS in ShangHai
```

Comment:
SuperProxy shouldn't located @ shanghai, write a simple socks5 server in golang locally to test it!!!!!!!!!!!!!!